src/sink/elastic.rs

```diff
@@ -198,6 +445,41 @@ impl Elastic {
         }
     }
 }
+
+/// task responsible to receive raw response data, build response event and send it off.
+/// keeps ownership of `EventIdGenerator` and more
+async fn response_task(
+    mut id_gen: EventIdGenerator,
+    origin_uri: EventOriginUri,
+    rx: Receiver<(LineValue, Cow<'static, str>)>,
+    insight_tx: Sender<sink::Reply>,
+) -> Result<()> {
+    loop {
+        match rx.recv().await {
+            Ok((data, port)) => {
+                let response_event = Event {
+                    id: id_gen.next_id(),
+                    data,
+                    ingest_ns: nanotime(),
+                    origin_uri: Some(origin_uri.clone()),
+                    ..Event::default()
+                };
+                eprintln!("Sending response {:?} via {}", &response_event, port);
```

Comment:
Do we need to priont on every event? If so can we use `error!` to allow filtering / redirecting

Reply:
same below

Reply:
yeah, those are stupid printf-debugging leftovers. gonna clean them up soonish

Reply:
Ja I do the same :D and then fail to see them